packages/theme-product/__mocks__/product.js

```diff
@@ -0,0 +1,63 @@
+function getProductJsonMock() {
```

Comment:
This is a fixture and should go in a `__fixtures__ `folder.

You can change it to `product.json` and just have the JSON object. Best way to convert this JS object into JSON is to copy this whole function into a browser console and copy the result... that will put `".."` around all the keys and save you some time.

Reply:
Thanks for the tip! I wasn't sure how to implement the right way.

- I changed mocks folder to `__fixtures__` and converted the JS into json file
- I used `import productJson ...` and removed all the unnecessary JSON methods for converting format. It looks much cleaner. 👍